src/page-header/title/useTitle.tsx

```diff
@@ -1,12 +1,9 @@
 import { useEffect } from 'react'
-import { useDispatch } from 'react-redux'
 
-import { updateTitle } from './title-slice'
+import { useUpdateTitle } from './TitleContext'
 
 export default function useTitle(title: string): void {
-  const dispatch = useDispatch()
+  const updateTitle = useUpdateTitle()
 
-  useEffect(() => {
-    dispatch(updateTitle(title))
-  })
+  useEffect(() => updateTitle(title))
 }
```

Comment:
seems like we should be able to remove this so that we we only have two hooks `useTitle` and `useUpdateTitle`